### PR TITLE
SelectBox: popup height should be calculated correctly for the first opening if the pageSize is equal to dataSource items count (refix T942881)

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: testing/testcafe/tests/editors/selectBox/popup.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    return createWidget('dxSelectBox', {
+      dataSource: {
+        store: [],
+        paginate: true,
+        pageSize: 3,
+      },
+    });
+  });
+
+  test(`SelectBox has a correct popup height for the first opening if the pageSize is less to dataSource length, theme=${theme}`, async (t) => {
```
Comment (ksercs) [inline testing/testcafe/tests/editors/selectBox/popup.ts]:
```suggestion
  test(`SelectBox has a correct popup height for the first opening if the pageSize is less than dataSource items count, theme=${theme}`, async (t) => {
```

File: testing/testcafe/tests/editors/selectBox/popup.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    return createWidget('dxSelectBox', {
+      dataSource: {
+        store: [],
+        paginate: true,
+        pageSize: 3,
+      },
+    });
+  });
+
+  test(`SelectBox has a correct popup height for the first opening if the pageSize is more to dataSource length, theme=${theme}`, async (t) => {
```
Comment (ksercs) [inline testing/testcafe/tests/editors/selectBox/popup.ts]:
```suggestion
  test(`SelectBox has a correct popup height for the first opening if the pageSize is more than dataSource items count, theme=${theme}`, async (t) => {
```

File: testing/testcafe/tests/editors/selectBox/toolbarIntegration.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import url from '../../../helpers/getPageUrl';
+import SelectBox from '../../../model/selectBox';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`SelectBox as Toolbar item`
+  .page(url(__dirname, '../../container.html'));
+
+test('SelectBox should correctly render his buttons when editor rendered as Toolbar item with fieldTemplate(T949859)', async (t) => {
```
Comment (ksercs) [inline testing/testcafe/tests/editors/selectBox/toolbarIntegration.ts]:
```suggestion
test('SelectBox should correctly render its buttons if editor is rendered as a Toolbar item with fieldTemplate(T949859)', async (t) => {
```

**Review by Zedwag — DISMISSED** [review]

**Review by ksercs — DISMISSED** [review]
Good job!

File: testing/testcafe/tests/editors/selectBox/toolbarIntegration.ts (hunk 3 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/selectBox/toolbarIntegration.ts]: https://github.com/DevExpress/DevExtreme/pull/21986/commits/778fa8fc9871bab376f4e82ae2754e636e00abb1

File: testing/testcafe/tests/editors/selectBox/popup.ts (hunk 2 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/selectBox/popup.ts]: https://github.com/DevExpress/DevExtreme/pull/21986/commits/778fa8fc9871bab376f4e82ae2754e636e00abb1

File: testing/testcafe/tests/editors/selectBox/popup.ts (hunk 1 above)
Comment (EugeniyKiyashko) [reply testing/testcafe/tests/editors/selectBox/popup.ts]: https://github.com/DevExpress/DevExtreme/pull/21986/commits/778fa8fc9871bab376f4e82ae2754e636e00abb1

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**